components/rating/stories/rating.stories.js

```diff
@@ -156,6 +156,20 @@ Focused.parameters = {
 	chromatic: { disableSnapshot: true },
 };
 
+/**
+ * For the emphasized variant, when a selected star is hovered, the color of the star is slightly darker.
+ */
```

Comment:
So I mentioned this already on the .test.js file comment, but let's make sure we update this as well, since both variants do get a subtle color change on hover. Thanks for adding documentation for this!